cmd/zed/root/command.go

```diff
@@ -23,42 +21,27 @@ querying, and orchestrating Zed data lakes.`,
 
 type Command struct {
 	charm.Command
+	lakeflags.Flags
```

Comment:
We don't usually embed flag structs, so I don't think we should embed this one.
```suggestion
	LakeFlags lakeflags.Flags
```